src/hooks/use-certificate-trust.ts

```diff
@@ -1,35 +1,22 @@
-import { useCallback, useEffect, useState } from 'react';
+import { useCallback } from 'react';
 import { useWindowListener } from 'src/hooks/use-window-listener';
-import { getIpcApi } from 'src/lib/get-ipc-api';
+import { useCheckCertificateTrustQuery } from 'src/stores/certificate-trust-api';
 
 /**
  * Custom hook that checks if the Studio CA certificate is trusted on the system
  * @returns A boolean indicating if the certificate is trusted
  */
 export function useCertificateTrust(): boolean {
-	const [ isTrusted, setIsTrusted ] = useState< boolean >( false );
+	const { data: isTrusted = false, refetch: checkCertificateTrust } =
+		useCheckCertificateTrustQuery();
```

Comment:
```suggestion
	const { data: isTrusted = false } = useCheckCertificateTrustQuery( undefined, {
		refetchOnFocus: true,
	} );
```
I suggest doing away with this file and replacing the `useCertificateTrust` calls with this ☝️ code.

Reply:
Was able to get rid of the file.